_es/tour/basics.md

```diff
@@ -6,4 +6,319 @@ discourse: false
 
 partof: scala-tour
 language: es
+
+num: 2
+next-page: unified-types
+previous-page: tour-of-scala
 ---
+
+En esta página, cubriremos lo básico de Scala.
+
+## Probando Scala en el navegador
+
+Puedes ejecutar Scala en tu navegador con ScalaFiddle.
+
+1. Ve a [https://scalafiddle.io](https://scalafiddle.io).
+2. Escribe `println("Hello, world!")` en el panel a la izquierda.
+3. Presiona el botón "Run". En el panel de la derecha aparecerá el resultado.
+
+Así, de manera fácil y sin preparación, puedes probar fragmentos de código Scala.
+
+Muchos ejemplos de código en esta documentación están integrados con ScalaFiddle, y así puedes probarlos directamente solo con pulsar el botón "Run".
+
+## Expresiones
+
+Las expresiones son sentencias computables.
+
+```tut
+1 + 1
+```
+
+Se puede ver el resultado de evaluar expresiones usando `println`.
+
+{% scalafiddle %}
+```tut
+println(1) // 1
+println(1 + 1) // 2
+println("Hello!") // Hello!
+println("Hello," + " world!") // Hello, world!
+```
+{% endscalafiddle %}
+
+## Valores
+
+Se puede dar un nombre al resultado de una expresión usando la palabra reservada `val`.
+
+```tut
+val x = 1 + 1
+println(x) // 2
+```
+
+Los resultados con nombre, como `x` en el ejemplo, son llamados valores. Referenciar un valor no lo vuelve a computar.
+
+Los valores no pueden ser reasignados.
+
+```tut:fail
+x = 3 // This does not compile.
```

Comment:
```suggestion
x = 3 // Producirá un error de compilación.
```

Reply:
Creo que tut igualmente va a mostrar el mensaje en ingles tambien

Reply:
No he usado `tut`. I trust you :-)